src/main/kotlin/com/github/eisermann/geminilauncher/mcp/McpServer.kt

```
@@ -0,0 +1,278 @@
+        exchange.responseBody.use { it.write(bytes) }
+    }
+
+    override fun dispose() {
+        activeSseConnections.forEach {
+            try {
+                it.close()
+                // Notify waiting threads
+                synchronized(it) {
+                    (it as Object).notifyAll()
+                }
+            } catch (e: Exception) {}
```

detekt (1.23.8)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)